resources/zib/zib-MedicationAdministration2.xml

```diff
@@ -0,0 +1,318 @@
+      </mapping>
+    </element>
+    <element id="MedicationAdministration.medication[x]">
+      <path value="MedicationAdministration.medication[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="MedicationAdministration.medication[x]:medicationReference">
+      <path value="MedicationAdministration.medicationReference" />
+      <sliceName value="medicationReference" />
+      <short value="AdministrationProduct" />
+      <definition value="The product taken or administered. This is usually medication. Food, blood products, aids and bandages do not strictly fall under the category of medication, but can be reported as well. &#xD;&#xA;&#xD;&#xA;In principle, this will be the prescribed product, but the administrator may substitute it by replacing the product with an equivalent product. For example: two 50mg tablets can be administered instead of one 100mg tablet." />
+      <alias value="ToedieningsProduct" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Medication" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-PharmaceuticalProduct" />
+      </type>
+      <mapping>
+        <identity value="MedicationAdministration2-v1.1.1-2020EN" />
+        <map value="NL-CM:9.13.20929" />
+        <comment value="AdministrationProduct" />
+      </mapping>
+    </element>
+    <element id="MedicationAdministration.supportingInformation">
+      <path value="MedicationAdministration.supportingInformation" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="type" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="MedicationAdministration.supportingInformation:administrationAgreement">
+      <path value="MedicationAdministration.supportingInformation" />
+      <sliceName value="administrationAgreement" />
+      <short value="AdministrationAgreement" />
+      <definition value="Administration agreement on which this administration is based." />
+      <alias value="Toedieningsafspraak" />
+      <max value="1" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-AdministrationAgreement" />
+      </type>
+      <mapping>
+        <identity value="MedicationAdministration2-v1.1.1-2020EN" />
+        <map value="NL-CM:9.13.23237" />
+        <comment value="AdministrationAgreement" />
+      </mapping>
+    </element>
+    <element id="MedicationAdministration.supportingInformation:administrationAgreement.type">
+      <path value="MedicationAdministration.supportingInformation.type" />
+      <min value="1" />
+      <fixedUri value="MedicationDispense" />

```

Comment:
This must be on a pattern not a fixed uri:

https://informatiestandaarden.nictiz.nl/wiki/FHIR:V1.0_FHIR_Profiling_Guidelines_R4#Slicing

 The sliced element gets a pattern or a required ValueSet binding on the location of the path.

Reply:
On the level of Primitive Types a fixed value is exactly similar as a pattern. For consistency, I made it a pattern.